actions/update-issue/action.yml

```diff
@@ -0,0 +1,102 @@
+name: 'Update TUF-on-CI issue'
+description: 'Create, close or add a comment in a GitHub issue for a workflow failure'
+# * This action will open an issue per workflow if that workflow fails.
+# * If an issue is open for that workflow already, the action will add a comment.
+# * If an issue is open and the workflow succeeds, the action will close the issue.
+# * The issue is identified using a label that is the workflow filename.
+# * Required permissions:
+#     issues: write
+
+inputs:
+  token:
+    description: 'GitHub token'
+    required: true
+
+  success:
+    description: '"true" if workflow is succeeding'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Update issue
+      uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+      env:
+        SUCCESS: ${{ inputs.success }}
+      with:
+        github-token: ${{ inputs.token }}
+        script: |
+          var path = require("path")
+
+          success = (process.env.SUCCESS == "true")
+
+          // Find issue labeled with the forkflow filename
+          const label = path.basename(context.payload.workflow)
```

Comment:
bug: context.payload.workflow is a path only if `name` is not set in the workflow. Need to figure out another useful label

Reply:
fixed by using the workflow name as label